subsys/logging/Kconfig

```diff
@@ -44,6 +44,19 @@ config LOG_DEFAULT_LEVEL
 	  - 3 INFO, default to write LOG_LEVEL_INFO
 	  - 4 DEBUG, default to write LOG_LEVEL_DBG
 
+config LOG_OUTPUT
+	int "Selected log output"
+	default 3
+	range 0 4
+	help
+	  Sets log output
+
+	  - 0 NONE
```

Comment:
if for some reason it is needed I think those should all be options instead of int value